### PR TITLE
initial pass at including docker-options plugin. closes #1062

### LINKED ISSUE #1062: Add dokku-docker-options as official plugin
Refs dyson/dokku-docker-options#6

### PR DESCRIPTION
**Pull request by michaelshobbs**
I basically copied what was in the `dokku_merge` branch. Feel free to comment.

@dyson @josegonzalez

### CODE REVIEW

**Comment by josegonzalez** [comment]
lol all the lint issues :P

File: docs/docker-options.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+-v /another/container/path
+```
+
+Remove an option
+```bash
+$ dokku docker-options:remove myapp "-link container_name:alias"
+```
+
+Manual Usage
+------------
```
Comment (josegonzalez) [inline docs/docker-options.md]: Lets make this `Advanced Usage` and have a note to users that they should avoid this workflow if at all possible.

File: plugins/docker-options/docker-args-build [hunk 2]
```diff
@@ -0,0 +1 @@
+docker-args-deploy
```
Comment (josegonzalez) [inline plugins/docker-options/docker-args-build]: Whats this do?

Comment (michaelshobbs) [inline plugins/docker-options/docker-args-build]: Must be how github renders a softlink?

File: docs/docker-options.md (hunk 1 above)
Comment (michaelshobbs) [inline docs/docker-options.md]: Done.

**Comment by josegonzalez** [comment]
Can you mark dyson's plugin as deprecated (there is a section on obsolete plugins) in the plugins doc as well? I think this is mergeable after that.

**Comment by michaelshobbs** [comment]
Done and done!

**Comment by josegonzalez** [comment]
Yay docker options! Hurray people screwing this up!

**Comment by michaelshobbs** [comment]
:+1:

**Pull request merged**